sql/mariadb/MariaDBLexer.g4

```diff
@@ -326,6 +326,9 @@ JSON_STORAGE_FREE:                   'JSON_STORAGE_FREE';
 JSON_STORAGE_SIZE:                   'JSON_STORAGE_SIZE';
 JSON_ARRAYAGG:                       'JSON_ARRAYAGG';
 JSON_OBJECTAGG:                      'JSON_OBJECTAGG';
+NESTED:                              'NESTED';
+ORDINALITY:                          'ORDINALITY';
+PATH:                                'PATH';
```

Comment:
Shouldn't they be added to [`keywordsCanBeId`](https://github.com/antlr/grammars-v4/blob/master/sql/mysql/Positive-Technologies/MySqlParser.g4#L2636)?